src/chainlit/__init__.py

```diff
@@ -232,4 +244,8 @@ def sleep(duration: int):
     "on_stop",
     "action_callback",
     "sleep",
+    "ChainlitCallbackHandler",
```

Comment:
Should this be changed to `SyncChainlitCallbackHandler`? This would make it easier to choose between the two.